joomla/joomla-cms
# Add utf8mb4 unicode conversion for com_finder

Pull Request for Issue #9361 .

### Summary of Changes
Convert `#__finder_xxx` tables to utf8mb4_unicode_ci collation


### Testing Instructions
See #9361

Upgrade joomla from version 3.4.8, 3.5.x or 3.7.2 still works.
Joomla should contain contents with words like:
* ausschließlich
* ausschliesslich

Details:
1. Install joomla 3.7.2 or staging
2. Enable plg_content_finder
3. Create article with problematic words mentioned above.
4. Install patch, for example by com_patchtester
5. Go to Extensions->Manage->Database
You should see message **"The Joomla! Smart Search database tables (in #__finder...) have not been converted yet to UTF-8 Multibyte (utf8mb4)."**
6. Click on fix button.
7. Database is OK.

### Expected result
#__finder_xxx tables has unicode collation.
Upgrade OK

### Actual result
Missing unicode collation in #__finder_xxx tables.


### Documentation Changes Required
I do not know.
